<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User-Centric Software to Assist Design for Forming</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="grid_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="3">
       <widget class="QRadioButton" name="springback_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Springback</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="strain_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Major Strain</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="material_label">
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLabel" name="selected_label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>No model selected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="process_label">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="load_mesh_button">
        <property name="text">
         <string>Load New Mesh</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTreeView" name="dir_tree">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="material_dropdown"/>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="thinning_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Thinning</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="process_dropdown"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="slider_label">
        <property name="text">
         <string>Slider Placeholder</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="3" colspan="3">
       <widget class="GLViewWidget" name="main_view" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16777214</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16777214</width>
     <height>24</height>
    </size>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_New_Mesh"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionOptimisation_Mode"/>
    <addaction name="actionSensitiity_Mode"/>
    <addaction name="action_developer"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionImport_New_Mesh">
   <property name="text">
    <string>Import New Mesh</string>
   </property>
  </action>
  <action name="actionOptimisation_Mode">
   <property name="text">
    <string>Optimisation</string>
   </property>
  </action>
  <action name="actionSensitiity_Mode">
   <property name="text">
    <string>Sensitiity Analysis</string>
   </property>
  </action>
  <action name="action_developer">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.opengl</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
